mantidproject/mantid
# Concepts file names normalised

ticket

http://trac.mantidproject.org/mantid/ticket/11321
## To test
1. check that the file names now are PascalCase universally
2. check that links between the concepts still work
3. Check that the index still works and looks good
